boards/Makefile.common

```diff
@@ -207,14 +224,14 @@ show-target:
 # Support rules
 
 target:
-	@mkdir -p $(TOCK_ROOT_DIRECTORY)target
+	@mkdir -p $(TARGET_PATH)
 
 # Cargo outputs an elf file (just without a file extension)
 %.elf: %
 	$(Q)cp $< $@
 
 %.bin: %.elf $(TOCK_ROOT_DIRECTORY)tools/sha256sum/target/debug/sha256sum
-	$(Q)$(OBJCOPY) --output-target=binary $< $@
+	$(Q)$(OBJCOPY) $(OBJCOPY_FLAGS) $< $@
```

Comment:
I don't like this change. This rule expects to create a `.bin` file. It requires this flag to do that. Objcopy can also do many other things. If there is a motivating example of when this specific flag needs to change, then maybe we could consider a variable like `OBJCOPY_ELF2BIN_FLAGS`, but otherwise I think this is complexity and indirection for no value.

Reply:
Thats a good point. Reason I added OBJCOPY_FLAGS was to be able to customize additional operations like -S or -g to strip all debug/comment sections, potentially rename sections or adjust LMA/VMA. Will '--output-target=binary $(OBJCOPY_FLAGS)' better express intent?